### ISSUE TITLE
Warning: SSR Client Hydration Error 

### ISSUE THREAD

**Issue by think-in-universe** [near/near-explorer#692]
**Problem**

When opening the home page of Explorer frontend in local development environment (http://localhost:3000/), the below server and client mismatch error was thrown

```
Warning: Prop `placeholder` did not match. Server: "Search for Account ID, Txn hash, Block hash, or Block height" Client: "Missing translationId: component.utils.Search.hint for language: ${ languageCode }"
    in input (created by FormControl)
    in FormControl (at Search.jsx:86)
    in WrappedTranslate (created by Context.Consumer)
    in Translate (at Search.jsx:84)
    in div (created by InputGroup)
    in InputGroup (at Search.jsx:73)
    in div (created by Row)
    in Row (at Search.jsx:72)
    in form (at Search.jsx:68)
    in Search (at Header.jsx:59)
    in div (created by Col)
    in Col (at Header.jsx:55)
    in div (created by Row)
    in Row (at Header.jsx:23)
    in div (created by Container)
    in Container (at Header.jsx:22)
    in Header (at _app.jsx:85)
    in div (at _app.jsx:84)
    in NetworkProvider (at _app.jsx:80)
    in LocalizeProvider (at _app.jsx:62)
    in _App
    in ErrorBoundary (created by ReactDevOverlay)
    in ReactDevOverlay (created by Container)
    in Container (created by AppContainer)
    in AppContainer
    in Root
```

This doesn't break the features in production, but is something we may need to look into.

**Comment by luixo** [comment]
We should check if this is fixed (probably will) after [i18n PR](https://github.com/near/near-explorer/pull/848) merge

**Comment by frol** [comment]
@shelegdmitriy Could you check it? If it is resolved, let's close this issue, indeed.

**Comment by shelegdmitriy** [comment]
> @shelegdmitriy Could you check it? If it is resolved, let's close this issue, indeed.

I'll check

**Comment by shelegdmitriy** [comment]
Resolved by #848
